src/main/java/uk/gov/hmcts/reform/demo/controllers/HelloWorldEndpoint.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.demo.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static org.springframework.http.ResponseEntity.ok;
+
+@RestController
+public class HelloWorldEndpoint {
+
+    @GetMapping
+    public ResponseEntity<String> helloWorld() {
```

Comment:
Is this clear to teams that they shouldn't delete it?
i.e. I added this comment in the CMC team PR:
https://github.com/hmcts/cmc-claim-store/pull/263/files#diff-7a539344c51609e77ea7e6822993678aR10

Reply:
always ahead of time 🏎 . will add a comment

Reply:
Worth making it a TODO comment

Reply:
its not a TODO comment though, it should always be there

Reply:
as of #30 no TODOs needed. Init script will automatically replace slugs in files and will be ready for new project